settings.sh

```diff
@@ -80,7 +80,7 @@ searchall() {
         declare -p allsettings >"$CFG_CACHE" 2>/dev/null
     fi
     export SIDEBARINDEX=0
-    CHOICE=$(for k in "${!allsettings[@]}"; do echo "$k"; done | sidebar)
+    CHOICE=`printf '%s\n' "${!allsettings[@]}" | sidebar`
```

Comment:
The backticks are of course an anachronism.

Reply:
It's valid syntax, but I do ordinarily try to stick to syntax preferred by the project, so I apologise for straying from your preferred `$()` form of command substitution. I hadn't used the graves as part of the optimization, it was just habit.